wiki/People/The_Team/Developers/fr.md

```diff
@@ -0,0 +1,53 @@
+# Développeurs
+
+Les développeurs d'osu! gardent le jeu et le site internet à jour en corrigeant les bugs et en ajoutant de nouvelles fonctionnalités. La plupart sont des employés de ppy Pty Ltd, l'entreprise à qui appartient osu!. Ils forment la plupart de la [osu!team](/wiki/People/The_Team).
+
+Les développeurs d'osu! peuvent être reconnu par leur couleur rose et les titres "osu!dev" ou "osu!team".
+
+## Membres du groupe
+
+La [page du groupe des développeurs](https://osu.ppy.sh/groups/11) liste tous les membres de l'équipe.
+
+Le développement d'osu! est dirigé par ![][flag_AU] **[peppy](https://osu.ppy.sh/users/2)**, le créateur d'osu!.
+
+### Développeur du client du jeu
+
+- ![][flag_FR] [Damnae](https://osu.ppy.sh/users/989377)
+- ![][flag_NO] [MillhioreF](https://osu.ppy.sh/users/941094)
+- ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+- ![][flag_AU] [smoogipoo](https://osu.ppy.sh/users/1040328)
+- ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058)
+
+### Développeur du site internet et du serveur
+
+- ![][flag_JP] [nanaya](https://osu.ppy.sh/users/2387883)
+- ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102)
+- ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776)
+- ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+
+### Designers
+
+- ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765)
+- ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202)
+
+### Community managers
```

Comment:
just making sure, this (and "Designers") is the common way to write it in french? there's no translation?

Reply:
We use english word for Community managers and Designer in French